src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java

```diff
@@ -71,6 +73,46 @@ public void adsLeaderboardAndInContentOnPage(String wikiName,
   @Test(
       groups = "AdsSlotsMercury",
       dataProviderClass = MobileAdsDataProvider.class,
+      dataProvider = "leaderboardSlotOnPageWithInfobox"
+  )
+  public void adsLeaderboardOnPageWithInfobox(String wikiName,
+                                               String article,
```

Comment:
Hey, what's wrong with this alingment? Is it about space before second parameter? I'm not familiar with this convention and I used it as it's on other tests in this class. Thanks!